### ISSUE TITLE
Subtitle text ignored within <>

### ISSUE THREAD

**Issue by chriskempson** [rsimmons/voracious#11]
I have subtitles like this:
> (坂東はな)<今から 70年ほど前>

When displayed in Voracious I see this:
> (坂東はな)

The text inside the `<` and `>` is probably being treated as an HTML tag.

**Comment by rsimmons** [comment]
Many subtitles have embedded HTML tags for formatting, and Voracious strips and ignores those using a simple regex. You are correct that your subs are getting mistakenly affected by that. I'm not sure what the easiest way to avoid this is, but I will take a look at it sometime.

**Comment by chriskempson** [comment]
Perhaps an allow list of html tags would be sufficient? I've noticed VLC is able to handle my subs but not sure how it does it.

**Comment by rsimmons** [comment]
Yeah that would work, I think there are only a handful that get used in subs.
